src/query/frontend/ast/cypher_main_visitor.cpp

```diff
@@ -283,12 +283,20 @@ antlrcpp::Any CypherMainVisitor::visitCreateEdgeIndex(MemgraphCypher::CreateEdge
   auto *index_query = storage_->Create<EdgeIndexQuery>();
   index_query->action_ = EdgeIndexQuery::Action::CREATE;
   index_query->edge_type_ = AddEdgeType(std::any_cast<std::string>(ctx->labelName()->accept(this)));
+  if (ctx->propertyKeyName()) {
+    auto name_key = std::any_cast<PropertyIx>(ctx->propertyKeyName()->accept(this));
+    index_query->properties_ = {name_key};
```

Comment:
Why properties are std::vector instead of only single property?

Reply:
The same thing is done for label-property indices. It could be probably preparation for indices with 2 or more properties.